WalletWasabi.Gui/Converters/InverseBooleanConverter.cs

```diff
@@ -10,17 +10,12 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 		{
 			if (value is bool boolean)
 			{
-				if (boolean)
-				{
-					return false;
-				}
-				else
-				{
-					return true;
-				}
+				return boolean ? false : true;
```

Comment:
This is just a comment: this converter an other are not really necessary because there is a much simpler way to implement the same `FuncValueConverter`. I didn't know about it before today. I will create an issue to improve this.

Reply:
Ok, I merge it now and deploy.